src/scripts/main.js

```diff
@@ -1,3 +1,25 @@
 'use strict';
 
-// write your code here
+const arrayPopulationStr = [
+  ...document.getElementsByClassName('population'),
+].map((n) => n.textContent);
```

Comment:
The use of the spread operator with getElementsByClassName is correct to convert the HTMLCollection to an array. This allows you to use array methods like map.